src/internal/dns/dns.go

```diff
@@ -0,0 +1,45 @@
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: 2021-Present The Zarf Authors
+
+// Package dns contains DNS related functionality.
+package dns
+
+import (
+	"errors"
+	"fmt"
+	"net/url"
+	"regexp"
+	"strconv"
+)
+
+var (
+	// localClusterServiceRegex is used to match the local cluster service format:
+	localClusterServiceRegex = regexp.MustCompile(`^(?P<name>[^\.]+)\.(?P<namespace>[^\.]+)\.svc\.cluster\.local$`)
+)
+
+// IsServiceURL returns true if the give url complies with the service url format.
```

Comment:
```suggestion
// IsServiceURL returns if the give url complies with the service url format.
```

Reply:
Why remove true? It returns either way but only true if the pattern matches?

Reply:
This is not typically how I see the comments for those that return bools, but if it's what you are used to feel free to ignore.